src/main/java/org/jenkinsci/plugins/workflow/steps/StepExecution.java

```diff
@@ -14,7 +14,7 @@
 import java.util.logging.Logger;
 import edu.umd.cs.findbugs.annotations.CheckForNull;
 import edu.umd.cs.findbugs.annotations.NonNull;
-import javax.inject.Inject;
+import jakarta.inject.Inject;
```

Comment:
In this context the Guice version is perhaps more appropriate, but I do not think it really matters.